wweval/R/supplement_analyses_and_figs.R

```diff
@@ -1058,3 +1058,87 @@ get_heatmap_metadata_hub <- function(metadata,
     )
   )
 }
+
+#' Get the overall relative crps for the real time scores
+#'
+#' @param all_scores a tibble of the scores for both models in real-time
+#'
+#'
+#' @return A tibble summarized the relative crps across all locs
+#' and forecast dates produced in real-time
+#' @export
+get_overall_rel_crps_rt <- function(all_scores) {
```

Comment:
```suggestion
get_overall_rel_crps_real_time <- function(all_scores) {
```

Reply:
This name is too many characters! I tried this earlier